.eslintrc.js

```diff
@@ -89,9 +89,12 @@ module.exports = {
 			rules: {
 				'import/no-nodejs-modules': 'off',
 				'no-console': 'off',
-				'jest/valid-describe': 'off',
-				'jest/no-test-prefixes': 'off',
-				'jest/no-identical-title': 'off',
+
+				// Disable all rules from "plugin:jest/recommended", as e2e tests use mocha
+				...Object.keys( require( 'eslint-plugin-jest' ).configs.recommended.rules ).reduce(
```

Comment:
I wonder if we could be specific here about the rules we're disabling. Although it's dynamic, this feels a bit fragile and dependant on what Jest considers "recommended".

Reply:
My intention is to disable all of them, always, even if Jest recommendation changes. The reason is these are not Jest at all, they use Mocha. So even if Jest recommends a new test X and for whatever reason it doesn't fail in Mocha tests, it will still be brittle to have it activated for our Mocha tests.

In other words, this is like disabling the eslint plugin `jest` for tests that don't use Jest, but `eslint` doesn't provide the option to enable/disable plugins for some paths, just rules.